docs/build/avalanchego-apis/info.md

```diff
@@ -232,6 +232,59 @@ curl -X POST --data '{
 }
 ```
 
+
+### info.getVMs
+
+Get the virtual machines installed on this node.
+
+```text
+info.getVMs() -> {
+    vms: map[string][]string
+}
+```
+
+#### **Example Call**
+
+```bash
+curl -X POST --data '{
+    "jsonrpc":"2.0",
+    "id"     :1,
+    "method" :"info.getVMs",
+    "params" :{}
+}' -H 'content-type:application/json;' 127.0.0.1:9650/ext/info
+```
+
+#### **Example Response**
+
+```javascript
```

Comment:
IMO these should be `json` but it would probably be inconsistent with other places.

Reply:
Updated